docs/deployment/containers/docker.rst

```diff
@@ -154,20 +154,16 @@ Let's add the second node, ``crate02``, to the cluster::
 
 Notice here that:
 
-- We updated the container and node name to ``crate02``
-- We updated the port mapping, so that port ``4202`` on your host is
-  mapped to ``4200`` on the container
-- We have set parameter ``discovery.seed_hosts`` to contain the other
-  hosts of the cluster
-- ``cluster.initial_master_nodes``:  In our scenario, we only want nodes
-  ``crate01`` and ``crate02`` to participate in the election of first
-  master, thus we leave it unchanged
-- ``gateway.expected_nodes`` and ``gateway.recover_after_nodes``: We have
-  three nodes in the cluster and we want the cluster's state to be recovered
-  once all nodes are started
-
-Now, if you go back to the admin UI you already have open, or visit the admin
-UI of the node you just created (located at ``http://localhost:4202/``) you
+- You updated the container and node name to ``crate02``
```

Comment:
Would these lists, including also further below, benefit from semicolons at the end (with final period)? Seems a bit more readable to me (but up to you)